packages/cli/src/commands/build.ts

```diff
@@ -366,6 +408,17 @@ export default async function main(client: Client): Promise<number> {
     );
   }
 
+  // Cleanup any globals remaining from corepack
```

Comment:
All of this cleanup shouldn't be necessary. We don't spawn and more child processes after this point in `vc build` and env vars don't "persist" after the lifetime of the Node process.